internal/addon/var.go

```diff
@@ -18,6 +18,18 @@ const (
 	AdcMetricsDisabledKey = "metricsDisabled"
 	AdcLoggingDisabledKey = "loggingDisabled"
 	AdcTracingisabledKey  = "tracingDisabled"
+
+	ClusterLogForwardersResource = "clusterlogforwarders"
+	SpokeCLFName                 = "instance"
+	SpokeCLFNamespace            = "openshift-logging"
+	clfProbeKey                  = "isReady"
+	clfProbePath                 = ".status.conditions[?(@.type==\"Ready\")].status"
+
+	OpenTelemetryCollectorsResource = "opentelemetrycollectors"
+	SpokeOTELColName                = "spoke-otelcol"
+	SpokeOTELColNamespace           = "spoke-otelcol"
+	otelColProbeKey                 = "replicas"
+	otelColProbePath                = ".spec.replicas"
```

Comment:
Why do we replicate the resource names here? Doesn't `loggingv1` provide one for `clusterlogforwarders`? Same question for `opentelemetrycollectors`

@JoaoBraveCoding @iblancasa Do we want to keep deploying instances of `opentelemetrycollector` on the spokes with the name `spoke-otelcol`? WDYT start referring them with a prefix like `mcoa-...`? Same for `clusterlogforwarders`

@JoaoBraveCoding Can the health prober work with CLFs/OtelCollectors named differently per clusterset? For example one creates a CLF resource called `mcoa-eu-clusters-instance` for a custom placement in the `ClusterManagementAddon`. How does probing work here?

Reply:
> @JoaoBraveCoding @iblancasa Do we want to keep deploying instances of opentelemetrycollector on the spokes with the name spoke-otelcol? WDYT start referring them with a prefix like mcoa-...? Same for clusterlogforwarders

Yes, it makes sense.
But, anyway, these values should be get from the AddonConfig not hardcoded.

Reply:
> I'm fine with calling them both `mcoa-instance` or anything else. @iblancasa At least for now, I don't see a use case where we would need to make these names configurable using `AddonConfig`.

I would prefer to keep it done from the `AddonConfig` since it's the "source of truth". Also... we will support, at least for the `OpenTelemetryCollector`, the creation of multiple instances. And those will be provided by users.